### PR TITLE
Added instalation of LLDP rules

### PR DESCRIPTION
**Pull request by niksv**
Part of #2582

### CODE REVIEW

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowConnectedDeviceSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+import groovy.util.logging.Slf4j
+import spock.lang.Narrative
+import spock.lang.Unroll
+
+@Slf4j
+@Narrative("Verify allocated Connected Devices resources and installed rules.")
+class FlowConnectedDeviceSpec extends HealthCheckSpecification {
+
+    def "Able to create flow with connected devices"() {
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowConnectedDeviceSpec.groovy]: This test is already covered by the below `Unroll` test, isn't it?

File: services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java [hunk 2]
```diff
@@ -197,13 +212,30 @@ long installTransitFlow(DatapathId dpid, String flowId, Long cookie, int inputPo
...
+     * @param dpid           datapathId of the switch
+     * @param flowId         flow id
+     * @param cookie         flow cookie
+     * @param inputPort      port to expect packet on
+     * @param outputVlanId   vlan to match on output vlan id
+     * @param meterId        flow meter ID
+     * @return transaction id
+     * @throws SwitchOperationException Switch not found
+     */
+    long installOneSwitchLldpFlow(DatapathId dpid, String flowId, long cookie, int inputPort, int outputVlanId,
```
Comment (timofei-durakov) [inline services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java]: this should be the same as lldp for ingress

Comment (niksv) [reply services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java]: fixed

File: services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java [hunk 3]
```diff
@@ -419,6 +425,65 @@ public long installIngressFlow(DatapathId dpid, String flowId, Long cookie, int
...
+        List<OFInstruction> instructions = new ArrayList<>();
+
+        if (meter != null) {
+            instructions.add(meter);
+        }
+
+        instructions.add(actions);
+
+        if (enableLldp) {
+            instructions.add(ofFactory.instructions().gotoTable(TableId.of(1)));
```
Comment (timofei-durakov) [inline services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java]: TABLE_1

Comment (niksv) [reply services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java]: fixed

**Pull request merged**